src/libcore/char.rs

```diff
@@ -136,230 +136,25 @@ pub fn from_digit(num: u32, radix: u32) -> Option<char> {
     }
 }
 
-/// Basic `char` manipulations.
-#[stable(feature = "rust1", since = "1.0.0")]
+// NB: the stabilization and documentation for this trait is in
+// unicode/u_char.rs, not here
```

Comment:
`src/libunicode/char.rs`, right? By the way, why was `u_char.rs` renamed `char.rs` but `u_str.rs` was left as-is?